src/libstd/sys/unix/backtrace/tracing/gcc_s.rs

```diff
@@ -53,7 +53,7 @@ pub fn unwind_backtrace(frames: &mut [Frame])
     // See libunwind:src/unwind/Backtrace.c for the return values.
     // No, there is no doc.
     match result_unwind {
-        uw::_URC_END_OF_STACK | uw::_URC_FATAL_PHASE1_ERROR => {
+        uw::_URC_END_OF_STACK | uw::_URC_FATAL_PHASE1_ERROR | uw::_URC_FAILURE => {
```

Comment:
I'm not ever sure how to comment this beyond "I don't care what these errors are, but these three need to be ignored for Travis to pass".